### PR TITLE
created blog

### PR DESCRIPTION
**Pull request by Scripore**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+gem 'builder', '~> 3.2.2'
+
+# Code syntax highlighting
+gem 'middleman-syntax', '~> 2.0.0'
+gem 'redcarpet', '~> 3.2.3'
+
+# For "summary"-Helper
+gem 'nokogiri', '~> 1.6.6.2'
+
+
```
Comment (nanoxd) [inline Gemfile]: Remove extra lines

File: README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+## Installation
+
+1. Fork this repo and then clone your repo to the desktop. `git clone <repo url>`
+
+
+## Usage
+
+Enter in the following commands
+```bash
+$ bundle install
```
Comment (nanoxd) [inline README.md]: No need to enter `$` when using backticks; causes issues with copy and pasting.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+$ git checkout source
+$ cd source/articles
+```
+
+Copy one of the existing markdown files and update the title, author, and date fields. Update the filename itself to correspond to the changes. Overwrite the blog post content with your own.
+
+Once finished, while still in the source branch, enter the following:
+```bash
+$ git add .
```
Comment (nanoxd) [inline README.md]: Same as above

File: README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+$ git commit -m '<enter in your commit message here>'
```
Comment (nanoxd) [inline README.md]: Avoid using the shorthand commit flag `-m`. It causes users to elongate the message. See [here](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) for more info

File: README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+$ git push
+```
+
+Once the changes are pushed to your repo, open a pull request to submit changes to main Operation Code blog repo.
```
Comment (nanoxd) [inline README.md]:
``` diff
- submit changes to main Operation Code blog
+ submit changes to the Operation Code blog
```

File: config.rb [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+  # blog.year_link = "{year}.html"
+  # blog.month_link = "{year}/{month}.html"
+  # blog.day_link = "{year}/{month}/{day}.html"
+  # blog.default_extension = ".markdown"
+
+  blog.tag_template = "tag.html"
+  # blog.calendar_template = "calendar.html"
+
+  # Enable pagination
+  blog.paginate = true
```
Comment (nanoxd) [inline config.rb:27]: Pagination is enabled but there is no hard `per_page` setting. What is the default?

File: config.rb [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+  deploy.method = :git
+  deploy.branch = 'master'
+  deploy.build_before = true
+end
+
+activate :directory_indexes
+
+set :casper, {
+  blog: {
+    url: 'http://www.example.com',
```
Comment (nanoxd) [inline config.rb]: URL is not set

File: config.rb [hunk 8]
```diff
@@ -0,0 +1,157 @@
...
+  proxy "/author/#{blog_author.name.parameterize}.html", '/author.html', ignore: true
+end
+
+###
+# Compass
+###
+
+# Change Compass configuration
+# compass_config do |config|
+#   config.output_style = :compact
```
Comment (nanoxd) [inline config.rb:83]: We should minify the CSS if it doesn't default to doing that.

File: config.rb [hunk 9]
```diff
@@ -0,0 +1,157 @@
...
+set :js_dir, 'javascripts'
+
+set :images_dir, 'images'
+
+set :partials_dir, 'partials'
+
+# Build-specific configuration
+configure :build do
+  # For example, change the Compass output style for deployment
+  # activate :minify_css
```
Comment (nanoxd) [inline config.rb]: Enable this

File: config.rb [hunk 10]
```diff
@@ -0,0 +1,157 @@
...
+
+  # Minify Javascript on build
+  # activate :minify_javascript
```
Comment (nanoxd) [inline config.rb]: And this

File: helpers/middleman_casper_helpers.rb [hunk 11]
```diff
@@ -0,0 +1,120 @@
...
+  end
+
+  def navigation
+    casper[:navigation]
+  end
+
+  def is_tag_page?
+    current_resource.metadata[:locals]['page_type'] == 'tag'
+  end
+  def tags?(article = current_article)
```
Comment (nanoxd) [inline helpers/middleman_casper_helpers.rb:62]: Spacing between above method

File: helpers/middleman_casper_helpers.rb [hunk 12]
```diff
@@ -0,0 +1,120 @@
...
+    article.tags.present?
+  end
+  def tags(article = current_article, separator = ', ')
```
Comment (nanoxd) [inline helpers/middleman_casper_helpers.rb:66]: Same

File: helpers/middleman_casper_helpers.rb [hunk 13]
```diff
@@ -0,0 +1,120 @@
...
+  end
+
+  def cover(page = current_page)
+    if (src = page.data.cover).present?
+      { style: "background-image: url(#{image_path(src)})" }
+    else
+      { class: 'no-cover' }
+    end
+  end
+  def cover?(page = current_page)
```
Comment (nanoxd) [inline helpers/middleman_casper_helpers.rb:87]: Same

File: helpers/middleman_casper_helpers.rb [hunk 14]
```diff
@@ -0,0 +1,120 @@
...
+    page.data.cover.present?
+  end
+
+  def gravatar(size = 68)
+    md5 = Digest::MD5.hexdigest(blog_author.gravatar_email.downcase)
+    "https://www.gravatar.com/avatar/#{md5}?size=#{size}"
+  end
+  def gravatar?
```
Comment (nanoxd) [inline helpers/middleman_casper_helpers.rb:96]: Same

File: helpers/middleman_casper_helpers.rb [hunk 15]
```diff
@@ -0,0 +1,120 @@
...
+    blog_author.gravatar_email.present?
+  end
+
+  def twitter_url
+    "https://twitter.com/share?text=#{current_article.title}" \
+      "&amp;url=#{current_article_url}"
+  end
+  def facebook_url
```
Comment (nanoxd) [inline helpers/middleman_casper_helpers.rb:105]: Same

File: helpers/middleman_casper_helpers.rb [hunk 16]
```diff
@@ -0,0 +1,120 @@
...
+    "https://www.facebook.com/sharer/sharer.php?u=#{current_article_url}"
+  end
+  def google_plus_url
```
Comment (nanoxd) [inline helpers/middleman_casper_helpers.rb:109]: Ditto

File: helpers/middleman_casper_helpers.rb [hunk 17]
```diff
@@ -0,0 +1,120 @@
...
+  end
+
+  def feed_path
+    if is_tag_page?
+      "#{current_page.url.to_s}feed.xml"
+    else
+      "#{blog.options.prefix.to_s}/feed.xml"
+    end
+  end
+  def home_path
```
Comment (nanoxd) [inline helpers/middleman_casper_helpers.rb:121]: Ditto

File: helpers/middleman_casper_helpers.rb [hunk 18]
```diff
@@ -0,0 +1,120 @@
...
+    "#{blog.options.prefix.to_s}/"
+  end
+  def author_path
```
Comment (nanoxd) [inline helpers/middleman_casper_helpers.rb:125]: Ditto

**Comment by nanoxd** [comment]
I made a few comments. Please fix it and squash your commits into one commit with a little more detail.

Thanks!

**Comment by Scripore** [comment]
Will do!

**Comment by davidmolina** [comment]
@Scripore looking good so far!

**Pull request merged**